### ISSUE TITLE
Enhancement Request: __version__ attritbute

### ISSUE THREAD

**Issue by VeNoMouS** [ad-m/python-anticaptcha#41]
Can you please add a ```__version__``` attribute to your code base , so i don't have to do something as crude as 

```python
if not hasattr(client, 'createTaskSmee'):
    raise RuntimeError("Please upgrade 'python_anticaptcha' via pip or download it from https://github.com/ad-m/python-anticaptcha")
```

https://www.python.org/dev/peps/pep-0396/#specification

**Comment by ad-m** [comment]
Yes!

**Comment by ad-m** [comment]
To be aware that Smee is experimental feature.

**Comment by VeNoMouS** [comment]
> To be aware that Smee is experimental feature.

Roger that :)

**Comment by ad-m** [comment]
Smee comments are welcome. I'm afraid:
- Smee reaction in case of higher load, especially for many users of this library,
- Smee's reaction due to some controversial use cases,
- problems in SSE support, because I did not want to introduce a new dependency and the implementation of the protocol is quite naive (but effective).

Issues fixed in 0.4.1 (see changelog https://python-anticaptcha.readthedocs.io/en/latest/changes.html#id1 ), so I am going to to close that issues.
